tds/routers/external.py

```diff
@@ -90,6 +96,19 @@ def create_publication(
     """
     with Session(rdb) as session:
         publication_payload = payload.dict()
+        publications = (
+            session.query(orm.Publication)
+            .filter(orm.Publication.xdd_uri == publication_payload["xdd_uri"])
+            .all()
+        )
+        if len(publications) != 0:
+            return Response(
+                status_code=status.HTTP_200_OK,
+                headers={
+                    "content-type": "application/json",
+                },
+                content=json.dumps({"id": publications[0].id}),
+            )
```

Comment:
What is the expected behavior if the payload matches the xdd_uri, but has a different description? In this case, the different description is silently ignored. Is that what we want?